doc/user/content/overview/_index.md

```diff
@@ -10,185 +10,84 @@ menu:
+and keeps results up to date for you as **incrementally updated materialized
+views**.
+
+<br>
+
+| ![streaming_analytics.gif](https://user-images.githubusercontent.com/23521087/233637537-4c2e5133-5a93-4d9d-9800-a0d4ba842c82.gif) |
+|:---------------------------------------------------------------------------------------------------------------------------------:|
+| *As an example, you can use Materialize to keep your dashboards automatically up to date.*                                                           |
+
+## Features
+
+Materialize combines all the desirable features of SQL databases and abstracts
+away all the hard parts required to build a streaming engine horizontally
+scalable, highly available and strongly consistent. To get up and running, all
+you need to know is **SQL**.
+
+### Incrementally updated views
+
+Materialized views are useful to avoid re-running heavy queries that run
+frequently against a database, and typically use caching to serve results
+faster. In traditional databases, materialized views often come with a long
+list of limitations and force you to make a compromise between the freshness of
+the results, the cost of refreshing the view and the complexity of the SQL
+statements you can use.
+
+Materialize supports incrementally updated materialized views that are **always
+fresh**, and allows a much broader set of SQL statements, like views over
+multi-way joins with complex aggregations. How? Under the hood, its engine is
+powered by [Timely](https://github.com/TimelyDataflow/timely-dataflow#timely-dataflow) and
 [Differential Dataflow](https://github.com/timelydataflow/differential-dataflow#differential-dataflow),
-purpose-built for low-latency applications. It lets you ask complex questions
-about your data using **SQL**, and incrementally maintains the results of these SQL queries as the underlying data changes.
+backed by many years of research and optimized for this exact purpose.
 
-## What does Materialize do?
+### Standard SQL support
 
-You might be wondering: why not just use your database's built-in functionality
-to perform these same computations? Your database often acts as if it's never
-been asked that question before, which means it can take a _long_ time to come
-up with an answer, each and every time you pose the query.
+You interact with Materialize like you would with any other database: using
+standard SQL. We implement a large fraction of PostgreSQL, and are actively
+working on supporting more built-in PostgreSQL functions, operators and data
+types, as well as extending our `pg_catalog` support.
 
-Materialize instead keeps the results of the queries and incrementally updates
-them as new data comes in. So, rather than recalculating the answer each time
-it's asked, Materialize continually updates the answer and gives you the
-answer's current state.
+With Materialize, you can build complex analytical workloads using **any type of
```

Comment:
should we link to our join documentation here?

Reply:
Ah, good catch!